src/SprykerEco/Yves/Easycredit/Processor/SuccessResponseProcessor.php

```diff
@@ -85,7 +88,7 @@ protected function addEasycreditExpense(QuoteTransfer $quoteTransfer): QuoteTran
         $expenseTransfer = new ExpenseTransfer();
         $expenseTransfer->setType(EasycreditConstants::EASYCREDIT_EXPENSE_TYPE);
         $expenseTransfer->setUnitNetPrice(0);
-        $expenseTransfer->setUnitGrossPrice($this->moneyPlugin->convertDecimalToInteger($quoteTransfer->getPayment()->getEasycredit()->getAnfallendeZinsen()));
+        $expenseTransfer->setUnitGrossPrice($this->moneyPlugin->convertDecimalToInteger($quoteTransfer->getPaymentOrFail()->getEasycreditOrFail()->getAnfallendeZinsenOrFail()));
```

Comment:
Is it safe to call `getAnfallendeZinsenOrFail`?
The same question goes for similar cases below.

Reply:
Added casting to necessary types